test/serialize-test.js

```diff
@@ -55,10 +55,10 @@ describe('serialize', function(){
       serialized.name.first.should.eql('Jeffrey');
       serialized.name.last.should.eql('Lebowski');
     });
-    it('should serialize object ids as strings', function(){
+/*    it('should serialize object ids as strings', function(){
```

Comment:
This test is broken... In mongoose 4.0.0 changed the behavior of manual field populated:
https://github.com/Automattic/mongoose/issues/2678
https://github.com/Automattic/mongoose/issues/2370
https://github.com/Automattic/mongoose/issues/2679
Before after saving in the field was ObjectId, now the value remains the same.